requirements.yml

```diff
@@ -0,0 +1,27 @@
+name: deepchem
```

Comment:
I fixed the version of soft requirements.
But, I think we should decide on how to  manage the version.